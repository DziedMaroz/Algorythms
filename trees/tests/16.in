10
16
15
3
2
1
