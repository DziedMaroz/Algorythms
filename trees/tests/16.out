10
3
2
1
16
15

